recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/gles-user-module.inc

```diff
@@ -3,13 +3,18 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 DESCRIPTION = "PowerVR GPU user module"
 LICENSE = "CLOSED"
 
+inherit pythonnative
```

Comment:
Why do we need this? If the recipe just depends on native Python then I can't see why we need to inherit